### ISSUE TITLE
Add relevant trove classifiers to setup.py

### ISSUE THREAD

**Issue by pklapperich** [minrk/findspark#23]
Adding classifiers will allow findspark's PyPI package to show if users filter based on different categories (such as python version support). [More Info](https://pypi.org/classifiers/). pip also uses these classifiers to ensure dependencies are met during installation.

**Comment by minrk** [comment]
Makes sense! PRs welcome. How does pip use classifiers to resolve dependencies?
